commerce_coordinator/apps/titan/pipeline.py

```diff
@@ -126,14 +126,18 @@ def run_filter(self, order_data, **kwargs):  # pylint: disable=arguments-differ
         order_payment_state = order_data['payment_state']
 
         if order_payment_state not in correct_order_payment_states:
-            return None  # Order does not need draft payment. Halt pipeline.
+            logger.debug('Order does not need draft payment. Halt pipeline.')
+            return PipelineCommand.HALT.value
         elif not recent_payment:
-            return {}  # Continue. Will generate a new draft payment.
+            logger.debug('No draft payment found. Continue pipeline...')
+            return PipelineCommand.CONTINUE.value
         elif recent_payment['state'] == PaymentState.PENDING.value:
-            return None  # Order does not need draft payment. Halt pipeline.
+            logger.debug('Order already in progress. Halt pipeline.')
+            return PipelineCommand.HALT.value
         elif recent_payment['state'] == PaymentState.FAILED.value:
-            return {}  # Continue. Will generate a new draft payment.
-        # Inform pipeline that a draft payment already exists:
+            logger.debug('Failed payment found. Continue pipeline...')
+            return PipelineCommand.CONTINUE.value
+        logger.debug('Draft payment exists. Add draft payment to pipeline.')
```

Comment:
Can we add a little more detail in logs that can help us search these logs later on? may be order_uuid or payment_number

Reply:
Added in [496fde7](https://github.com/edx/commerce-coordinator/pull/115/commits/496fde7a8b8f0109027afba8223980495b00f4bb).